### ISSUE TITLE
Give a more descriptive title to 'Edit' button of several blocks

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#1689]
Several of our blocks have an editable view that can be accessed with the pen icon:
![imatge](https://user-images.githubusercontent.com/3616980/73461963-54be0880-437b-11ea-8ef9-65225493d7f0.png)

In all blocks, the title of that icon is _Edit_, but it doesn't behave the same in all blocks. In some of them it allows re-arranging inner blocks (_All Products_) and in some others it allows you to change the selected product, category, tag, etc.

Ideally, the title should be more descriptive so you don't need to toggle the button to know what it does.

**Comment by gigitux** [comment]
After my investigation, you can find in the following table which block has the edit button and my proposal for the new label.


| Block Name                   | Has Edit Icon | New Label                  |
| ---------------------------- | ------------- | -------------------------- |
| All reviews                  | 🚫            |                            |
| Featured Category            | ❎             | Edit selected category   |
| Featured Product             | ❎             | Edit selected product    |
| Hand-picked Products         | ❎             | Edit selected products   |
| Best Selling Products        | 🚫            |                            |
| Product Categories List      | 🚫            |                            |
| Products by Category         | ❎             | Edit selected categories |
| Newest Products              | 🚫            |                            |
| On Sale Products             | 🚫            |                            |
| Products by Attribute        | ❎             | Edit selected attributes |
| Top Rated Products           | 🚫            |                            |
| Reviews by Product           | ❎             | Edit selected product    |
| Reviews by Category          | ❎             | Edit selected categories |
| Product Search               | 🚫            |                            |
| Products by Tag              | ❎             | Edit selected tags       |
| All products                 | ❎             | ~~Rearrange inner blocks~~ Edit inner product layout                 |
| Filter products by Price     | 🚫            |                            |
| Filter Products by Attribute | 🚫            |                            |
| Filter Products by Stock     | 🚫            |                            |
| Active Product Filters       | 🚫            |                            |
| Checkout                     | 🚫            |                            |
| Card                         | 🚫            |                            |
| Single Product               | ❎             | Edit selected product      |
| Mini Cart                    | 🚫            |                            |
| Product Title                | 🚫            |                            |
| Product Price                | 🚫            |                            |
| Product Image                | 🚫            |                            |
| Product Rating               | 🚫            |                            |
| Add to Cart Button           | 🚫            |                            |
| Product Summary              | 🚫            |                            |
| On Sale Badge                | 🚫            |                            |
| Product SKU                  | 🚫            |                            |
| Product Category List        | 🚫            |                            |
| Product Tag List             | 🚫            |                            |
| Product Stock Indicator      | 🚫            |                            |
| Add to Cart                  | 🚫            |                            |

@Aljullu What do you think?

**Comment by Aljullu** [comment]
> @Aljullu What do you think?

Sounds good to me. Much better than what we have now! :slightly_smiling_face:
